addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+# Package for kpack
+
+[kpack](https://github.com/pivotal/kpack) utilizes unprivileged kubernetes primitives to provide builds of OCI images as a platform implementation of [Cloud Native Buildpacks](https://buildpacks.io) (CNB).
+
+## Components
+
+* kpack
+
+## Supported Providers
+
+The following tables shows the providers this package can work with.
+
+| GKE | AKS | EKS | vSphere | Minikube |
```

Comment:
I'm not sure  - do we only need to list the providers supported by TCE here - AWS, vSphere, Azure, Dockere?